### PR TITLE
Borrower

### PR DESCRIPTION
**Pull request by bmasles**
Added the borrower, implemented disabled button when fields aren't filled in

### CODE REVIEW

File: src/app/components/borrower/borrower.component.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      </div>
+      <div class="ui-g-8">
+        <input pInputText id="phone" [(ngModel)]="borrower.phone" autofocus />
+      </div>
+    </div>
+  </div>
+  <p-footer>
+    <div class="ui-dialog-buttonpane ui-helper-clearfix">
+      <button *ngIf="!newBorrower" type="button" pButton icon="pi pi-times" (click)="delete()" label="Delete"></button>
+      <button type="button" pButton icon="pi pi-check" (click)="save()" label="Save" [disabled]="!(borrower?.name && borrower?.address && borrower?.phone)"></button>
```
Comment (allistergg) [inline src/app/components/borrower/borrower.component.html]: are address and phone necessary?

File: src/app/components/borrower/borrower.component.ts [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  borrowers: Borrower[];
+  cols: any[];
+
+  constructor(private borrowerService: BorrowerService) { }
+
+  ngOnInit() {
+    this.borrowerService.getBorrowers().then(borrowers => {
+      return (this.borrowers = borrowers);
+    });
+    this.cols = [
```
Comment (allistergg) [inline src/app/components/borrower/borrower.component.ts]: don't you need address and phone columns here?

File: src/app/components/borrower/borrower.component.html (hunk 1 above)
Comment (bmasles) [reply src/app/components/borrower/borrower.component.html]: Shoud be fixed

File: src/app/components/borrower/borrower.component.ts (hunk 2 above)
Comment (bmasles) [reply src/app/components/borrower/borrower.component.ts]: Shoud be fixed

**Pull request merged**